kbi/0024/index.rst

```diff
@@ -0,0 +1,263 @@
+.. index::
+   single: RIA; archive
+.. highlight:: console
+
+KBI0024: Create and update a 7-Zip archive in a RIA store
+=========================================================
+
+:authors: Laura Waite <laura@waite.eu>, Michael Hanke <michael.hanke@gmail.com>
+:discussion: <link>
+:keywords: RIA, archive, 7-Zip, inodes
+:software-versions: datalad_0.18.3
+
+When working with `RIA stores`_, it is possible to compress the
+``annex/objects`` into `7-zip`_ archives. With this approach, the entire
+annex object store can be put into an archive and remain fully accessible while
+minimizing inodes, regardless of file number and size. This is beneficial
+for compression gains or when operating on HPC-systems with inode limitations.
+
+This document describes how to create a 7-Zip archive for an existing RIA
+store using the script shown below. The script does the following:
+
+1. Remove unnecessary files and directories within the RIA store (e.g. hooks,
+   etc). This aspect has nothing to do with creating the archive, but is useful
+   for reducing the number of inodes.
```

Comment:
```suggestion
1. Remove non-essential files and directories within the RIA store (e.g. hooks,
   etc). This aspect has nothing to do with creating the archive, but is useful
   for reducing the number of inodes.
```

To me _unnecessary_ begs the question why they are there in the first place, and non-essential sounds better. But maybe it's just me - in any case, this suggestion is non-essential :)